Sources/TuistCore/Models/Requirement.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+import XcodeProj
+
+public enum Requirement: Equatable {
```

Comment:
Since this is no longer namespace under `Package` it might be appropriate to give it a new name

Reply:
You're right. I'll go with `PackageRequirement`